### DISCUSSION TITLE
Feature

### DISCUSSION THREAD

**Discussion by Leigh-S** [Leigh-S/Tafe24S2T2EXT01#1] [General]
Are you happy if I proceed with the Mortgage Calculator development?

**Comment by Leigh-S** [comment]
With the style, I was just going to stick with the theme in the appendix. Is that okay with everyone?

**Reply by 000347334** [reply]
> With the style, I was just going to stick with the theme in the appendix. Is that okay with everyone?

I am happy with the appendix theme, it looks and works fine for now, we can "spice" up the UI later once the back end is solid.

**Comment by Leigh-S** [comment]
I haven't added the code behind the 'Main Menu' page yet. Is everyone happy to add code to redirect to their feature. Or would you like me to add skeleton code to avoid possible conflicts later?

**Reply by 000347334** [reply]
> I haven't added the code behind the 'Main Menu' page yet. Is everyone happy to add code to redirect to their feature. Or would you like me to add skeleton code to avoid possible conflicts later?

Do as much as you have time for, I wont be changing any class names, so any changes I make should be independent.

**Comment by 000347334** [comment]
> Are you happy if I proceed with the Mortgage Calculator development?

I am more than happy for you to work on the Mortgage Calculator.

**Comment by Leigh-S** [comment]
I have now merged and pushed the mortgage calculater feature.
